### PR TITLE
Add generator preferences and test the template

### PR DESCRIPTION
**Pull request by schpet**

### CODE REVIEW

File: rails-kickoff-template.rb [hunk 1]
```diff
@@ -40,13 +40,18 @@ def run_template!
...
   setup_readme
   create_database
 
   fix_bundler_binstub
 
+  setup_webpacker
+
   output_final_instructions
+
+  exit
```
Comment (travisp) [inline rails-kickoff-template.rb]: why did you add exit here? there is an additional step if this is being run on an existing application that this is skipping

File: rails-kickoff-template.rb [hunk 2]
```diff
@@ -113,24 +118,22 @@ def setup_bullet
...
+    1) Setup the Skylight ENV variable
+    2) Configure Sentry
+    3) Add the jemalloc buildpack:
+      $ heroku buildpacks:add --index 1 https://github.com/gaffenyc/heroku-buildpack-jemalloc.git
+    4) Setup Redis (if using Sidekiq)
+    5) Review your README.md file for needed updates
+    6) Review your Gemfile for formatting
+  MSG
+
+  say msg, :magenta
```
Comment (travisp) [inline rails-kickoff-template.rb]: any reason you removed this from after_bundle? won't this now run before bundle completes and you won't see the message without scrolling up?

File: test/test_getting_started.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    a = `gem install rails --no-document`
+    puts "which: #{`which rails`}"
+    puts "rails version: #{`rails --version`}"
+  end
+
+  def test_build_succeeds
+    assert(system("rm -rf tmp; mkdir -p tmp"), "tmp dir is replaced")
+
+    Dir.chdir("tmp") {
+      cmd = "yes | YES_ALL=1 rails new Foobar -T --skip-coffee --webpack --database=postgresql -m '../rails-kickoff-template.rb'"
```
Comment (travisp) [inline test/test_getting_started.rb]: using `yes` is generally fine, but for this question on line 413 `yes?("Is this template being run on an existing application? (usually no)")` The typical use of the application is to say no here. I agree it's a confusingly worded question and happy to get suggestions for better wording that might result in a yes default.

There are two ways to use this template: 1) to create a new application, 2) to apply to an existing application:

https://guides.rubyonrails.org/rails_application_templates.html

> You can use the app:template Rake task to apply templates to an existing Rails application.

This lets us apply new defaults to existing applications. It obviously can't be done blindly (every change will need to be reviewed), but my hope is it'll let us keep our default approach on applications a little more standard even as we update defaults.

**Comment by schpet** [comment]
@travisp thanks for looking at this and the helpful feedback! i've cleaned things up, using env vars instead of `$ yes` to automatically accept things, and i no longer need to `exit` or do anything weird.

tests pass locally for me:
```
Finished in 103.482756 seconds.
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
3 tests, 4 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
0.03 tests/s, 0.04 assertions/s
```

and i'm talking with semaphore so they should pass on CI soon hopefully.
